README.md

```diff
@@ -38,6 +38,72 @@ If you want to test what happens when the HTTP server goes down, use `Bypass.dow
 TCP socket and `Bypass.up/1` to start listening on the same port again. Both functions block until
 the socket updates its state.
 
+### Expect Functions
+
+You can take any of the following approaches:
+* `expect/2` or `expect_once/2` to install a generic function that all calls to bypass will use
+* `expect/4` and/or `expect_once/4` to install specific routes (method, path combinations)
+* a combination of the above, where the routes will be used first, and then the generic version
+  will be used as default
+
+#### expect/2 (bypass_instance, function)
+
+Must be called at least once, and it will error if not.  As an alternative, `nil` can be passed
```

Comment:
I was trying to keep things consistent with this:

https://github.com/PSPDFKit-labs/bypass/blob/master/test/bypass_test.exs#L80-L86

For backwards compatibility, don't you think it should be left in?  And if it works for `expect`, like before, then it seems like it would be confusing to *not* do it for `expect_once`, even if it would behave identically.  Thoughts, @alco ?